packages/react-composites/src/composites/CallWithChatComposite/adapter/CallWithChatBackedCallAdapter.ts

```diff
@@ -48,6 +48,9 @@ export class CallWithChatBackedCallAdapter implements CallAdapter {
   constructor(callWithChatAdapter: CallWithChatAdapter) {
     this.callWithChatAdapter = callWithChatAdapter;
   }
+  onReactionClicked(emoji: string): Promise<void> {
+    throw new Error(`Reaction of type ${emoji} send not successful.`);
```

Comment:
Is there a reason that we are throwing here? believably we need to make sure that we are able to use reactions in the CallWithChatComposite?